phpstan.neon

```diff
@@ -56,9 +56,18 @@ parameters:
         # PHPStan false positive? I don't think there is some constant expression
         -
             message: '#Move constant expression to __construct\(\), setUp\(\) method or constant#'
-            path: src/Rector/Class_/LatteVarTypesBasedOnPresenterTemplateParametersRector.php
+            paths:
+                - src/Rector/Class_/LatteVarTypesBasedOnPresenterTemplateParametersRector.php
+                - src/Rector/Class_/TemplateTypeBasedOnPresenterTemplateParametersRector.php
+
+        # PHPStan false positive. There is no method configure on reported line
+        -
+            message: '#Method configure\(\) with passed value object must contain assert to verify passed type#'
+            path: src/Rector/Class_/TemplateTypeBasedOnPresenterTemplateParametersRector.php
 
         # this is the way how it works in rector-src
```

Comment:
Should we change it here? https://github.com/rectorphp/rector-src/blob/main/src/NodeFactory/ClassWithPublicPropertiesFactory.php#L28

Reply:
Yes, the array is very crappy to work with. Should be value object instead

Reply:
Ok but it will be BC break right? Or it is not a problem?

Reply:
I think the `TemplateTypeBasedOnPresenterTemplateParametersRector` is our internal rule, so we can change anything there.

Also on 0.x BC breaks are allowed.

Reply:
I ment to change the ClassWithPublicPropertiesFactory in rector-src. I think I am the only one who uses it in rector-nette but who knows :)

Reply:
Yes :slightly_smiling_face: , go for it :+1:

Reply:
Can you please check this PR without the change in rector-src? Currently I am not able to do this refactoring.

Reply:
Sure :+1: